habitat-lab/habitat/config/habitat/task/rearrange/rearrange_easy_humanoid.yaml

```diff
@@ -0,0 +1,48 @@
+# @package habitat.task
```

Comment:
Same comment as above. I think we should not create another config for another agent. Can you compose configs or use argument overrides to achieve this?

Reply:
I started working on this, but I don't know how to remove the arm_action for the humanoid, that appears here: https://github.com/facebookresearch/habitat-lab/blob/1622862c53017376f6f7760c549e08b7efd49870/habitat-lab/habitat/config/habitat/task/rearrange/rearrange_easy.yaml#L45.

Even if I do override with the actions, arm_action appears and fails with the humanoid. Any idea to address this?